bin/redis-tls

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-
-# Switch to using rediss:// instead of redis://
-REDIS_URL="$(echo $REDIS_URL | sed 's/^redis:/rediss:/g')"
-
-# We need to pull out the port and increment it by one.
-REDIS_PORT="$(echo $REDIS_URL | cut -d ':' -f 4)"
-REDIS_PORT=$((REDIS_PORT + 1))
-
-# Replace the PORT in the URL with the incremented one.
-REDIS_URL="$(echo $REDIS_URL | sed "s/:[0-9][0-9]*$/:$REDIS_PORT/g" )"
-
-# Unset our temporary REDIS_PORT variable since we've now moved it into the
-# REDIS_URL.
-unset REDIS_PORT
-
-# Ensure all clients have a 5-second timeout.
-# Configure the TLS settings for our Redis connection
-# When adding parameters such as socket_timeout,
-# you may need to update the broker url parsing in
-# warehouse/tasks.py to apply them to celery_transport_options
-# See https://docs.celeryq.dev/projects/kombu/en/stable/_modules/kombu/connection.html#Connection
-# and https://docs.celeryq.dev/projects/kombu/en/stable/reference/kombu.transport.redis.html#transport-options
-# for available/allowed arguments/mappings
-REDIS_URL="$REDIS_URL?socket_timeout=5&ssl_cert_reqs=required&ssl_ca_certs=$(python -m certifi)"
```

Comment:
`ssl_ca_certs=$(python -m certifi)` was the main utility here, to get a newer CA cert than came with some past distro/base-image. this is no longer necessary as the trust root for Elasticache redis is now available in out base image.